index.bs

```diff
@@ -266,12 +287,14 @@ The <dfn export>distance fraction</dfn> of a {{Document}} |D| is the lesser of
     distance</a>, or
 * 1.0.
 
-The <dfn export>impact region</dfn> of a {{Document}} |D| is the set containing
+The <dfn export>node impact region</dfn> of a {{Node}} |N| is the set containing
 
-* every point in the <a>visual representation</a> of any {{Node}} in the
-    <a>unstable node set</a> of |D|, and
-* every point in the <a>previous frame visual representation</a> of any {{Node}}
-    in the <a>unstable node set</a> of |D|.
+* every point in the <a>visual representation</a> of |N|, and
+* every point in the <a>previous frame visual representation</a> of |N|.
+
+The <dfn export>impact region</dfn> of a {{Document}} |D| is the set containing
```

Comment:
nit: the naming seems a bit confusing because a Document has both a `node impact region` and an `impact region`. Perhaps rename the latter to something like `tree impact region` (or other name)?

Reply:
I prefer "node impact region" to "tree impact region" since it does not necessarily include the visual representations of its descendants.  However I have clarified that "node impact region" is only defined for an unstable node.  The document is never an unstable node.